utility/BUILD.bazel

```diff
@@ -0,0 +1,53 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+package(default_visibility = ["//visibility:public"])
+
+kt_android_library(
+  name = "utility_lib",
+  custom_package = "org.oppia.util",
+  srcs = [
+      "src/main/java/org/oppia/util/accessibility/*.kt",
+      "src/main/java/org/oppia/util/caching/*.kt",
+      "src/main/java/org/oppia/util/data/*.kt",
+      "src/main/java/org/oppia/util/datetime/*.kt",
+      "src/main/java/org/oppia/util/gcsresource/*.kt",
+      "src/main/java/org/oppia/util/logging/*.kt",
+      "src/main/java/org/oppia/util/networking/*.kt",
+      "src/main/java/org/oppia/util/parser/*.kt",
+      "src/main/java/org/oppia/util/profile/*.kt",
+      "src/main/java/org/oppia/util/system/*.kt",
+      "src/main/java/org/oppia/util/threading/*.kt",
+      "src/main/java/org/oppia/util/extensions/*.kt",
+      "src/main/java/org/oppia/util/statusbar/*.kt",
+  ],
+  resource_files = glob(["src/main/res/**/*.xml", "src/main/res/**/*.png"]),
```

Comment:
Does ``utility`` have any images?

Reply:
Also, whenever we have multiple items in a list we should list them on separate lines (I think this follows conventional formatting for Bazel BUILD files since it's generally Pythonic in formatting).

Reply:
Fixed!